snap/info.go

```diff
@@ -282,12 +285,17 @@ func (s *Info) CommonDataHomeDir() string {
 	return filepath.Join(dirs.SnapDataHomeGlob, s.Name(), "common")
 }
 
-// UserXdgRuntimeDir returns the XDG_RUNTIME_DIR directory of the snap for a particular user.
+// UserXdgRuntimeDirGlobal returns the toplevel XDG_RUNTIME_DIR directory for a particular user.
+func (s *Info) UserXdgRuntimeDirGlobal(euid int) string {
+	return filepath.Join("/run/user", fmt.Sprintf("%d", euid))
+}
+
+// UserXdgRuntimeDir returns the XDG_RUNTIME_DIR snap subdirectory for a particular user.
 func (s *Info) UserXdgRuntimeDir(euid int) string {
 	return filepath.Join("/run/user", fmt.Sprintf("%d/snap.%s", euid, s.Name()))
 }
 
-// XdgRuntimeDirs returns the XDG_RUNTIME_DIR directories for all users of the snap.
+// XdgRuntimeDirs returns the XDG_RUNTIME_DIR snap subdirectories for all users of the snap.
```

Comment:
We might take the chance to fix that one as well. Both the doc and the actual function name should mention that this is a glob.